Add a README to the CLI distribution package (#286)
wso2/micro-integrator: cmd/docs/README.md

> # WSO2 Micro Integrator Command Line tool (MI-CLI)

Reviewer: ```suggestion
# WSO2 Micro Integrator Command Line tool
```

> View and manage the micro integrator runtime using the “mi” command line tool. Some of the usages of the command line tool include,

Reviewer: ```suggestion
You can view and manage the Micro Integrator instances using the “mi” command line tool. Some of the usages of the command line tool include,
```

> 1. Get a list of deployment runtime artifacts

Reviewer: ```suggestion
1. Get a list of deployment runtime artifacts.
```

2. Inspect details of each runtime artifact such as a proxy service or an API.

> 3. Get the invocation endpoint of an artifact

Reviewer: ```suggestion
3. Get the invocation endpoint of an artifact.
```

### Running

> Extract the compressed archive generated to the desired location.

Reviewer: ```suggestion
1. Extract the compressed archive generated to the desired location.
```

Author: Since we pack the README inside the archive, this step was removed in cdbd7df

> Add the MI CLI bin folder to PATH in unix-based Operating System (Linux, Solaris and Mac OS X)

Reviewer: ```suggestion
2. Add the MI CLI bin folder to PATH in UNIX-based Operating System (Linux, Solaris, and Mac OS X)
```

> `export PATH=/path/to/mi/cli/directory/bin:$PATH`

Reviewer: ```suggestion
    `$ export PATH=/path/to/mi/cli/directory/bin:$PATH`
```

> Then execute, 

Reviewer: ```suggestion
4. Then execute, 
```

> `$ mi`

Reviewer: ```suggestion
    `$ mi`
```

Execute mi --help for further instructions.

### Add Command Autocompletion

> `$ source /path/to/mi/cli/directory/mi_bash_completion.sh`

Reviewer: Shall we also mention that this is only available for UNIX based operating systems?

### Configuration

> ##### Q. How to Enable the Management API

Reviewer: ```suggestion
##### How to Enable the Management API
```

By default the Management API is disabled. To use the Management API you must use the system property `-DenableManagementApi` when starting the micro integrator

> **NOTE: These APIs are not protected, do not enable this in production. Take extra measures to secure this port if you are enabling this in production.**

Reviewer: ```suggestion
**NOTE: These APIs are not protected using an authorization mechanism. Therefore take extra measures to secure this port if you are enabling this in production.**
```

> ##### Q. Management API Address and Port

Reviewer: ```suggestion
##### Management API Address and Port
```

To configure the address and the port of the Management API in the CLI use the [**init**](#init) command. This will generate a file called server_config.yaml which contains the address and the port. If the init command was not used, the address and the port will have the default values

> NOTE: The default address is https://localhost and the port is 9164

Reviewer: ```suggestion
NOTE: The default hostname is localhost and the port is 9164.
```
